js/spellChecker/spellChecker.js

```diff
@@ -0,0 +1,57 @@
+const { remote } = require('electron');
+const { MenuItem } = remote;
+const { isMac } = require('./../utils/misc');
+const { SpellCheckHandler, ContextMenuListener, ContextMenuBuilder } = require('electron-spellchecker');
+
+class SpellCheckHelper {
+
+    constructor() {
+        this.spellCheckHandler = new SpellCheckHandler();
+    }
+
+    /**
+     * Method to initialize spell checker
+     */
+    initializeSpellChecker() {
+        this.spellCheckHandler.attachToInput();
+
+        // This is only for window as in mac the
+        // language is switched w.r.t to the current system language.
+        //
+        // In windows we need to implement RxJS observable
+        // in order to switch language dynamically
+        if (!isMac) {
+            this.spellCheckHandler.switchLanguage('en-US');
```

Comment:
@KiranNiranjan Do we really need to validate any other language apart from the System set language? I don't think so.